data/hai/hai reveal 5 philosophers.txt

```diff
@@ -100,14 +100,14 @@ mission "Hai Reveal [B02] Join the Fleet"
+			`	"There's going to be a battle, and you have a critical role entrusted to you for coordinating the fleet. There can be no detours. Once we start moving it will be one jump to the rendezvous and then two to our target. Are you ready for it?"`
 			choice
 				`	"I'm ready now."`
 				`	"I need time to prepare."`
 					goto prepare
-			`	"Very good." So far she hasn't actually introduced herself but as she puts the pad away again her flight jacket moves aside enough to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups there have been positively identified as being involved in smuggling or kidnappings in the past. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress for many of those that live there. As such an agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective will be to overwhelm the defenders we're expecting to encounter, prevent any escapes, and occupy the planet while they ferret out all the groups intelligence has flagged.`
-			`	"The Navy, Unfettered, and Hai fleets will travel from Mountaintop to meet you. We must fly with all haste directly to Sumar to meet them where you will act as the central coordination vessel. My security force here will follow you there, but until we get to Sumar don't worry yourself about waiting for us. We'll handle ourselves. Once we're all together in Sumar, it'll be your job to make sure everyone moves as one for each jump from there to <system>."`
-			`	She lowers her voice even further, despite the privacy afforded the conversation already. "There will be two Navy Carrier groups, our full Syndicate Special Security Detachment, the Unfettered Expeditionary Fleet, and a full Hai Taskforce. I don't think I've ever heard of a fleet this large being in the one place, at least not since the Alpha Wars. This coordination role is not a joke. There's lots of scope for things to go wrong here. We will go in, clean up, and search for Hai. Parliament will reassess the scope of concern later depending on what we find down there. We follow your lead Captain <last>."`
+			`	"Very good." So far she hasn't introduced herself but as she puts the pad away, her flight jacket moves enough to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups intelligence has flagged.`
```

Comment:
```suggestion
			`	"Very good." So far she hasn't introduced herself, but as she puts the pad away her flight jacket moves enough for you to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups Intelligence has flagged.`
```

Reply:
"Intelligence" with a capital "I" doesn't make sense here because it implies there is an organization called "Intelligence" that collected it. With a lower-case "i," it means an unspecified government intelligence-gathering organization has flagged it. That is correct usage, albeit an obscure one. I'll rewrite that sentence after I have time to ponder it.

Reply:
Actually, come to think of it, keeping the original lower-case "i" intelligence makes the most sense because it is a special ops marine speaking. She'd use special ops jargon, so "intelligence" makes the most sense than spelling out the name of the intelligence agency (which she's probably not allowed to do anyway).

Reply:
There is an organisation called Intelligence under the Republic.

Reply:
All right. 
```suggestion
			`	"Very good." So far she hasn't introduced herself, but as she puts the pad away her flight jacket moves enough for you to see 'Captain Valeria Estrada' printed on the jumpsuit underneath. "Now, to bring you up to speed: At least two pirate groups at <planet> have been involved in smuggling or kidnappings. <planet> is a fortress of anonymity and discretion, as well as being a literal fortress. An agreed upon force of overwhelming numbers has been prepared at Mountaintop. The objective is to overwhelm the defenders, prevent any escapes, and occupy the planet while they ferret out all the groups intelligence has flagged.`
```